index.js

```diff
@@ -0,0 +1,87 @@
+const express = require('express');
+const cors = require('cors');
+const knex = require('knex');
+const knexConfig = require('./knexfile.js')
+const bcrypt = require('bcryptjs');
+const session = require('express-session');
+const secret = "secret";
+const KnexSessionStore = require('connect-session-knex')(session);
```

Comment:
Make sure you remove unneeded dependencies from your code. You shouldn't need `express-session` and `connect-session-knex` in this lab.